### PR TITLE
:bug: None Type in cvss score in Trivy #9263

### LINKED ISSUE #9263: NoneType Conversion error while reimporting trivy scan results
**Bug description**
Reimporting trivy findings via api fails.
Looks like this behaviour is about pr #9114 .

**Steps to reproduce**
Reimport scan results with `SeveritySource: nvd` but without CVSSV3 informations like:
```
{
  "VulnerabilityID": "CVE-2013-7445",
  "PkgID": "linux-libc-dev@6.1.55-1",
  "PkgName": "linux-libc-dev",
  "InstalledVersion": "6.1.55-1",
  "Status": "will_not_fix",
  "Layer": {
    "Digest": "sha256:938cae48a646a95127345a544f75f4e0b83f5fa612858e524aedea6981af4a1e",
    "DiffID": "sha256:dfe25755ef07309fcb76dd47b2bb21e6dd92adedce8d9aa7f5bbceaf7fc726c9"
  },
  "SeveritySource": "nvd",
  "PrimaryURL": "https://avd.aquasec.com/nvd/cve-2013-7445",
  "DataSource": {
    "ID": "debian",
    "Name": "Debian Security Tracker",
    "URL": "https://salsa.debian.org/security-tracker-team/security-tracker"
  },
  "Title": "kernel: memory exhaustion via crafted Graphics Execution Manager (GEM) objects",
  "Description": "The Direct Rendering Manager (DRM) subsystem in the Linux kernel through 4.x mishandles requests for Graphics Execution Manager (GEM) objects, which allows context-dependent attackers to cause a denial of service (memory consumption) via an application that processes graphics data, as demonstrated by JavaScript code that creates many CANVAS elements for rendering by Chrome or Firefox.",
  "Severity": "HIGH",
  "CweIDs": [
    "CWE-399"
  ],
  "VendorSeverity": {
    "nvd": 3,
    "redhat": 2,
    "ubuntu": 2
  },
  "CVSS": {
    "nvd": {
      "V2Vector": "AV:N/AC:L/Au:N/C:N/I:N/A:C",
      "V2Score": 7.8
    },
    "redhat": {
      "V2Vector": "AV:N/AC:M/Au:N/C:N/I:N/A:P",
      "V2Score": 4.3
    }
  },
  "References": [
    "https://access.redhat.com/security/cve/CVE-2013-7445",
    "https://bugzilla.kernel.org/show_bug.cgi?id=60533",
    "https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2013-7445",
    "https://lists.freedesktop.org/archives/dri-devel/2015-September/089778.html (potential start towards fixing)",
    "https://nvd.nist.gov/vuln/detail/CVE-2013-7445",
    "https://www.cve.org/CVERecord?id=CVE-2013-7445"
  ],
  "PublishedDate": "2015-10-16T01:59:00.12Z",
  "LastModifiedDate": "2015-10-16T16:22:25.587Z"
}
```
In the mentioned pull request we assume that, when `SeveritySource: nvd` is found in the vulnerability object there must be a CVSSV3 information which is might be missing.

from `dojo/tools/trivy/parser.py`:
```
severity_source = vuln.get("SeveritySource", None)
cvss = vuln.get("CVSS", None)
cvssv3 = None
if severity_source is not None and cvss is not None:
    cvssclass = cvss.get(severity_source, None)
    if cvssclass is not None:
        severity = self.convert_cvss_score(cvssclass.get("V3Score", None))
        cvssv3 = dict(cvssclass).get("V3Vector", None)
```

**Expected behavior**
A successful reimport.

**Deployment method**
- [ ] Docker Compose
- [X] Kubernetes
- [ ] GoDojo

**Environment information**
 - DefectDojo version: 2.30.0

**Logs** 
```
[03/Jan/2024 10:27:26] ERROR [dojo.api_v2.exception_handler:36] float() argument must be a string or a real number, not 'NoneType'
Traceback (most recent call last):
  File "/usr/local/lib/python3.11/site-packages/rest_framework/views.py", line 506, in dispatch
    response = handler(request, *args, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/rest_framework/mixins.py", line 19, in create
    self.perform_create(serializer)
  File "/app/dojo/api_v2/views.py", line 3238, in perform_create
    serializer.save(push_to_jira=push_to_jira)
  File "/app/dojo/api_v2/serializers.py", line 2502, in save
    ) = reimporter.reimport_scan(
        ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/app/dojo/importers/reimporter/reimporter.py", line 604, in reimport_scan
    parsed_findings = parser.get_findings(scan, test)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/app/dojo/tools/trivy/parser.py", line 90, in get_findings
    return self.get_result_items(test, results, artifact_name=artifact_name)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/app/dojo/tools/trivy/parser.py", line 176, in get_result_items
    severity = self.convert_cvss_score(cvssclass.get("V3Score", None))
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
--
[03/Jan/2024 10:27:26] ERROR [django.request:241] Internal Server Error: /api/v2/reimport-scan/
```

### PR DESCRIPTION
**Pull request by manuel-sommer**
Fix #9263 

- Makes None Type in CVSS Score to "Info Finding"
- Adds CVSSv2

### CODE REVIEW

File: dojo/tools/trivy/parser.py [hunk 1]
```diff
@@ -55,17 +55,20 @@ def get_description_for_scan_types(self, scan_type):
         return "Import trivy JSON scan report."
 
     def convert_cvss_score(self, raw_value):
-        val = float(raw_value)
-        if val == 0.0:
+        if raw_value is None:
```
Comment (manuel-sommer) [inline dojo/tools/trivy/parser.py:58]: Should we adapt the raw_value== None to score "Low"?

Comment (mtesauro) [reply dojo/tools/trivy/parser.py:58]: We have "Info" which feels like a better fit than "Low" for a raw_value == None

Comment (grendel513) [reply dojo/tools/trivy/parser.py:58]: in my opinion None should not equal Low as it misrepresents the risk associated with the finding.  If the tool has not assigned a value, then we should likely interpret that as Informational.

Comment (manuel-sommer) [reply dojo/tools/trivy/parser.py:58]: Ok, then I will leave it like it is. As soon as a severity is available, the old one will be closed anyway as severity is also in the deduplication settings in settings.dist.py

**Review by grendel513 — APPROVED** [review]

**Review by mtesauro — APPROVED** [review]
Approved

**Review by Maffooch — APPROVED** [review]

**Review by cneill — APPROVED** [review]

**Pull request merged**